picard/ui/metadatabox/__init__.py

```diff
@@ -138,13 +139,24 @@ def get_tag_name(self, index):
 
 class MetadataBox(QtWidgets.QTableWidget):
 
+    MimeConverters = namedtuple('MimeConverters', ('encode_func', 'decode_func'))
+
     MIMETYPE_PICARD_TAGS = "application/vdr.picard"
     MIMETYPE_TSV = 'text/tab-separated-values'
     MIMETYPE_TEXT = 'text/plain'
     mimedatas = {
-        MIMETYPE_PICARD_TAGS: 'to_json',
-        MIMETYPE_TSV: 'to_tsv',
-        MIMETYPE_TEXT: 'to_tsv',
+        MIMETYPE_PICARD_TAGS: MimeConverters(
```

Comment:
Maybe we should add a comment about the order in this dict, since it impacts paste operation (each are tried until first success).